ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -234,9 +233,9 @@ module.exports = Marionette.LayoutView.extend({
     const queryContentView = this.queryView
       ? this.queryView
       : this.queryContent.currentView
-    if (!queryContentView.isValid()) {
-      announcement.announce(InvalidSearchFormMessage)
-      return
+    const validation = queryContentView.validate()
```

Comment:
✏️ don't need another `validation` could just pass it in directly -> `if (!validate(queryContentView.validate()))`
applies to every time this is called